disnake/message.py

```diff
@@ -245,6 +245,11 @@ class Attachment(Hashable):
     ----------
     id: :class:`int`
         The attachment's ID.
+    title: Optional[:class:`str`]
+        The attachment title.
```

Comment:
```suggestion
        The attachment title. If the filename contained special characters,
        this will be set to the original filename, without filename extension.
```